### DISCUSSION TITLE
Will the different inputs of different training steps affect the model effect?

### DISCUSSION THREAD

**Discussion by vincezengqiang** [hpcaitech/ColossalAI#3631] [Community | General]
hihi thanks for the excellent work!
I found that the sftdataset will  rewrite by prompt_dict，and the reward model is trained by raw  data， and  in input of inference.py just like：'Question: How are you ? Answer:'。 
so  i  m curious that will difference inputs of different training steps affect the model effect？
Looking forward to your reply, thanks!

**Comment by binmakeswell** [comment]
Hi @vincezengqiang Sure, dataset is crucial to the model performance. This is the reason why some people say "models are data-centric"
